assets/logo.svg

```diff
@@ -0,0 +1,29 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve"
```

Comment:
Why were the assets moved around and modified?

Reply:
Adding the icon to the root directory didn't look good.. Too many assets. So, I intended to move them into an assets directory similar to nushell's structure.. 

The logo.svg was a one liner while the logo-dark/light were multi line files.. I had run a formatter on all of them.

Reply:
But yes, I could limit the scope of this PR to the main build file and the required dependencies..
Shall I revert the 2nd commit?

Reply:
The file is a one liner since the newlines aren't necessary and only increase the file size (the file was minified)

icon.ico can be placed into contrib/, alongside AppImage files that are already there

Reply:
Ahh I see, will make the changes